experiments/functional/csi-volume-resize/test.yml

```diff
@@ -79,6 +107,30 @@
               delay: 10
               retries: 50
 
+             ## Here we will dump +1Gi data than to previous pvc size
+            - set_fact:
+                value_num: '{{ ( (value_str.stdout | int + 1 | int) * 1024) |  int }}'
+
+            - name: Dump some more dummy data in the application mount point for using resized volume when application is busybox
+              shell: >
+                  kubectl exec -it "{{ app_pod.stdout }}" -n "{{ app_ns }}" 
+                  -- sh -c "cd {{ mount.stdout }} && dd if=/dev/urandom of=volume.txt bs=1024k count={{ value_num }}"
+              args:
+                  executable: /bin/bash
+              register: load
+              failed_when: "load.rc != 0"
+              when: "mount.stdout == '/busybox'"
+
+            - name: Dump some dummy data in the application mount point for using resized volume when application is percona
```

Comment:
@w3aman why there is two same tasks for dumping the data

Reply:
for busybox and percona, we have diff mount points so for that i added two diff task seperated by when condition

Reply:
In this tasks mount point is not hard coded anywhere we were obtaining the mount point and then only we are using son only one task is enough i thing